scripts/cc-cli/src/commands/wizard.ts

```diff
@@ -83,7 +84,7 @@ export function makeWizardCommand() {
     console.log("Using the following parameters:");
     console.log(`💰 Stash account: ${stashAddress}`);
     console.log(`🕹️  Controller account: ${controllerAddress}`);
-    console.log(`🪙  Amount to bond: ${amount} CTC`);
+    console.log(`🪙  Amount to bond: ${toCTCString(amount)}`);
```

Comment:
Can we remove the leading space before `Amount` & `Controller account` lines ?

Reply:
We should, but at least for me those emojis are weird and occupy half the space that the others do. Does it look different on other OSs or terminals?

Reply:
For me (Linux+bash) all emojis appear the same. It may be down to shell/font preferences though. 

I wouldn't block this PR on the number of white spaces.